app/src/main/java/com/mk/tasky/core/data/remote/exceptions/TaskyException.kt

```diff
@@ -0,0 +1,3 @@
+package com.mk.tasky.core.data.remote.exceptions
+
+class TaskyException(message: String) : Exception(message)
```

Comment:
What kind of exception is this? Could you tell by the name?

Reply:
You are right. Going to rename it to NetworkErrorException
Thanks!